### ISSUE TITLE
Issue reading `request.body.sender.username`

### ISSUE THREAD

**Issue by gmurphey** [npm/npm-hook-receiver#1]
While using npm-hook-slack to test out webhooks, I was getting 500s from my endpoint, and it seems to come from this package, or in the webhook trigger service itself. Here's the response from endpoint when receiving a trigger:

``` json
{
    "code": "InternalError",
    "message": "Cannot read property 'username' of undefined"
}
```

I believe this comes from https://github.com/npm/npm-hook-receiver/blob/master/index.js#L33. The request body from the trigger doesn't contain a `sender.username`. If you'd like the body from the received trigger, I'm happy to provide it, and I'm happy to file a PR after getting some input on how this should be handled. Thanks!

**Comment by soldair** [comment]
oh thanks! we switched last minute to hookOwner because sender was too confusing

**Comment by soldair** [comment]
published to npm v0.0.7 =) thanks!
